tutorials/23_MultiHop_QA_With_Agents.ipynb

```diff
@@ -0,0 +1,5680 @@
+            "39d8323f93734fd3828f487e7f3b3b15",
+            "a2eb26464c0343439fd34c13a75dd226",
+            "4023555fc4744ca3abf3ae4faf453ba0",
+            "17984128558e4daabec0e30bfac2ba7f",
+            "a688af77dc7c4fc0990cf0cd0c35e4ce",
+            "637f612921e149f39052eea7056d6dda",
+            "2100ec9a329c448b9f3cbc69c0a42702",
+            "307c067efe1a40bf83b7a9fde587246c",
+            "da47da32f07f4150986c8bf0ba894bdf",
+            "e5df7d5057224543988b1c647275fc99",
+            "bb256ee0baf6437484ff0c98fc9fba4b",
+            "3938942c91554ac39d467615d0ecf3f8",
+            "818644d41841422a8bb172d8042f1ee3",
+            "006efd706beb452cb087e4c1a937a3e7",
+            "e26386dae946438a940f1785668dac9b",
+            "e8b2b114a5c74a9c9bd71f75f4181c57",
+            "e78d0169f4004ddfb10d5ad4b30ba214",
+            "2c757f8401db440daccd830fb834dc39",
+            "0789ac89edc040219afa4515963c38e6",
+            "37d0759be9da45b99d21805c68f43e45",
+            "d889cde02e7e4e70823734bc28f60afd",
+            "ab1c6e33cb6c4730968d363e1dd16fd8",
+            "8dfb9f4386c547688eaea79b1f5d4de6",
+            "8bd107882cd14fc498229ba72043a3a8",
+            "7602886a31b94e5dbfd5bbd19969f178",
+            "dcebe2a2d0964a28aac1ce80a6f528d8",
+            "19ebf3419f044e80b091691a4774b1df",
+            "68614e5c7f4a4973a5c6990e8ea80f6a",
+            "695868cd0d644e0badf11f793af53c2d",
+            "fa0bf4ca5f9b406082a7d1a6dccb32b5",
+            "e22644ca58294d4dbcddd208583886b9",
+            "272225718b18418f9d9832662abdca00",
+            "3d85d0886e1f4e2fb9b812e89fa34bd8",
+            "c594db0163e64685929954ce4b4c0a6e",
+            "896b831869b24f3bb06a611375e0feea",
+            "110e84e5a4344d27b750e272e24ca523",
+            "d6e19afe43c640e894ca47f9a38fbc60"
+          ]
+        },
+        "id": "n9bmrIUCatMk",
+        "outputId": "9f110081-72ce-4f80-fea0-10b3f8a61654"
+      },
+      "outputs": [],
+      "source": [
+        "from haystack.nodes import BM25Retriever, FARMReader\n",
+        "from haystack.pipelines import ExtractiveQAPipeline\n",
+        "\n",
+        "retriever = BM25Retriever(document_store=document_store)\n",
+        "reader = FARMReader(model_name_or_path=\"deepset/roberta-base-squad2\", use_gpu=True)\n",
+        "presidents_qa = ExtractiveQAPipeline(reader=reader, retriever=retriever)\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mgQAP-4T7Fh6"
+      },
+      "source": [
+        "### 3) Ask a question!\n",
+        "\n",
+        "Now that you have an ExtractiveQAPipeline, go ahead and ask it a question about the presidents of the USA. \n",
+        "\n",
+        "Below is the question: \"Who was the 44th president of the USA?\"\n",
+        "\n",
+        "To start seeing the limitations of an ExtractiveQAPipeline on it's own, you may want to try a question like: \"Who is the wife of the 44th president of the USA?\".\n",
```

Comment:
```suggestion
        "To see the limitations of an ExtractiveQAPipeline on its own, try a question like: \"Who is the wife of the 44th president of the USA?\".\n",
```

(that's because of our voice: be conscise :) )